app/assets/javascripts/discourse/models/user-draft.js.es6

```diff
@@ -0,0 +1,60 @@
+import RestModel from "discourse/models/rest";
+import { on } from "ember-addons/ember-computed-decorators";
+import computed from "ember-addons/ember-computed-decorators";
+import { postUrl } from "discourse/lib/utilities";
+import { userPath } from "discourse/lib/url";
+
+import {
+  NEW_TOPIC_KEY,
+  NEW_PRIVATE_MESSAGE_KEY
+} from "discourse/models/composer";
+
+export default RestModel.extend({
+  @on("init")
+  _attachCategory() {
+    const categoryId = this.get("category_id");
+    if (categoryId) {
+      this.set("category", Discourse.Category.findById(categoryId));
+    }
+  },
+
+  @computed("draft_username")
+  editableDraft(draft_username) {
+    return draft_username === Discourse.User.currentProp("username");
```

Comment:
`import User from "discourse/models/user";`

You can use the above to gain access to the `User` class instead of accessing it via the global.